types/setting.go

```diff
@@ -94,6 +97,7 @@ const (
 	SettingNameFailedBackupTTL                                          = SettingName("failed-backup-ttl")
 	SettingNameRecurringSuccessfulJobsHistoryLimit                      = SettingName("recurring-successful-jobs-history-limit")
 	SettingNameRecurringFailedJobsHistoryLimit                          = SettingName("recurring-failed-jobs-history-limit")
+	SettingNameRecurringJobMaxRetain                                    = SettingName("recurring-job-max-retain")
```

Comment:
Retain is verb, let's use `retention` instead. Please change the var name and description.